src/Components/ApiRequest/ApiRequest.js

```diff
@@ -0,0 +1,11 @@
+const ApiResults = location => {
+  const api = Object.freeze({
+    API_KEY: '&appid=0d26436ded0fcb43c6227b80b9ae1a69',
+    API_URL: 'http://api.openweathermap.org/data/2.5/weather?q=',
+  });
+  const data = `${api.API_URL}${location}${api.API_KEY}`;
+
+  return data;
+};
+
+export default ApiResults;
```

Comment:
Api URL shoud be stored under directory 
./src/common/variables

const API_URL = 'url'

**The whole file is for deletion**

Reply:
I'll do in in .env file